src/libraries/Common/src/System/Security/Cryptography/Asn1/AsnXml.targets

```diff
@@ -1,16 +1,59 @@
 <Project>
   <PropertyGroup>
     <AsnXml />
-    <_AsnXmlDiffCmd>%24%28command -v diff%29 -q -a</_AsnXmlDiffCmd>
-    <_AsnXmlDiffCmd Condition="$([MSBuild]::IsOsPlatform('windows')) == 'true'">$(SystemRoot)\System32\fc.exe /a</_AsnXmlDiffCmd>
-    <_AsnXmlDiffCmd Condition="$([MSBuild]::IsOsPlatform('osx')) == 'true'">diff</_AsnXmlDiffCmd>
```

Comment:
FWIW, another option is to use appropriate switches to ignore CR: `fc.ex /a /w` and `diff --strip-trailing-cr -U1`

Reply:
Another cross-platform option is to use `git -c "core.autocrlf=true" diff --no-index` for out-of-tree file comparison. However, it'd fail if someone is building source build tarball on a machine without `git` installed on a system (e.g. a container); which seems unlikely for any reasonable build machine and we may just skip the comparison if `command -v git` (or `where git` on Windows) result is empty with a warning.

Reply:
We've had a number of issues opened in the past because people build without Git available and I think we did address that so people can build the source tarball/zip (eg https://github.com/dotnet/runtime/issues/47130). So I didn't want to take a further dependency on Git.

Skipping the diff entirely if there is no git is certainly an option, but I didn't see a big reason to do it that way.